server.js

```diff
@@ -651,20 +651,59 @@ proxyForAuditing.on('error', function(err, req, res) {
   serverUtils.serverError(JSON.stringify(err), req, res);
 });
 
-var couchDbVersionCheck = function(callback) {
-  db.getCouchDbVersion(function(err, version) {
-    console.log('CouchDB Version:', version);
+const nodeVersionCheck = callback => {
+  try {
+    console.log('Node Version:', process.version);
+
+    const version = process.versions.node.match(/(\d)+\.(\d)+\.(\d)+/)[1];
+
+    if (Number(version[1] <= 4)) {
```

Comment:
Might be nice to be consistent and say `< 5` here, as per the comment on the following line.  Also, the code below also uses `<` instead of `<=`.